packages/devtools_app/integration_test/test_infra/run/_test_app_driver.dart

```diff
@@ -199,6 +197,10 @@ class TestDartCliApp extends IntegrationTestApp {
     : super(appPath, TestAppDevice.cli);
 
   static const vmServicePrefix = 'The Dart VM service is listening on ';
+  static const controlPortKey = 'controlPort';
+
+  late final int? _controlPort;
+  int? get controlPort => _controlPort;
```

Comment:
style nit: order of getters and setters https://github.com/flutter/devtools/blob/master/STYLE.md#order-of-getters-and-setters

Reply:
Done.